airbyte-cdk/java/airbyte-cdk/s3-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/s3/jsonschema/AirbyteJsonSchemaType.kt

```diff
@@ -175,7 +177,7 @@ enum class AirbyteJsonSchemaType {
                 // Usually the root node
                 return OBJECT_WITH_PROPERTIES
             } else {
-                throw IllegalArgumentException("Unspecified schema type")
```

Comment:
Should we be explicit about the fact that we're matching an empty `{}`? Or do you want to use this as a catch-all

Reply:
more a catchall. So, I guess I should rename it to `unknown`  in that case 🤔